apps/api/src/report/types/duration-entry.ts

```diff
@@ -0,0 +1,8 @@
+export interface DurationEntry {
+  stage?: string;
+  milestone?: string;
+  field: string;
+  Mean?: number;
+  Mode?: number;
+  Median?: number;
```

Comment:
how come these are capitalized?

Reply:
I don't remember exactly. My guess is it's to avoid selectively capitalizing field names when generating spreadsheets.